### PR TITLE
Redirect query param

### PR DESCRIPTION
**Pull request by ActiveChooN**
### Motivation and context

This PR provide redirect parameter for auth page for redirecting after sucessfull login.

### How has this been tested?

Manually

### Test scenarios

1. Create some task if not existed
2. Logout and go to ``/tasks/<task_id>`` it will redirect to ``/auth/login?next=/tasks/<task_id>``
3. Login
4. Check correct task is opened

Using token login add parameter next manually: ``/login-with-token/:sessionId/:csrfToken?next=/tasks/<task_id>``

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat-ui/src/components/login-with-token/login-with-token.tsx [hunk 1]
```diff
@@ -24,7 +26,7 @@ export default function LoginWithTokenComponent(): JSX.Element {
     );
 
     if (cookies.sessionid && cookies.csrftoken) {
-        return <Redirect to='/tasks' />;
+        return <Redirect to={redirectParam || '/tasks'} />;
```
Comment (tdowgiel) [inline cvat-ui/src/components/login-with-token/login-with-token.tsx:29]: Redirection works fine at first time. 
Unfortunately when user is already loged in {cookies are set} It does not redirect to requested page any more

Comment (bsekachev) [reply cvat-ui/src/components/login-with-token/login-with-token.tsx]: Please, try another implementation according to updated PR description.

Comment (tdowgiel) [reply cvat-ui/src/components/login-with-token/login-with-token.tsx]: Hi, Thanks for the update. 
Tested on Chrome 87 - macOS (Big Sur) and Windows 10
Unfortunatle issue mentioned before still persists:

Steps to reproduce:
1. 
Action:
Login to CVAT using ?nexto fo example  http:/<ip>/auth/login-with-token/aj52qlbiux7manf8xwkz9etz1ivg8atk/nR9CeVzrL0AZMh8ZuAEzBr6swCeWUVbWT6KfScBbNuCMUW4dU1tU50tKb7ofUp9B?next=/tasks/1
Expecteted result: 
CVAT is logged and page is redirected to /tasks/1
2.
Action:
Open new browser tab and past link once again:
Expected result:
CVAT is logged and page is redirected to **/tasks/1**
Actual result:
CVAT is logged but page is redirected to **/tasks**

Regards

Comment (bsekachev) [reply cvat-ui/src/components/login-with-token/login-with-token.tsx]: @tdowgiel 
I thought you told about login with UI elements (twice or more) with the redirection query parameter. Anyway, looks like we fixed also login with a token.

**Review by nmanovic — APPROVED** [review]

**Pull request merged**